### DISCUSSION TITLE
GE Meeting May 15 (May 1 is a holiday)

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#122] [General]
Dear all,

May 1st is a holiday in Spain (and some other places), so I suppose let's skip that one.

So the next one will be May 15th, 18:30 London time. I will send a reminder! We will use Zoom as Teams works poorly with Linux. 

I will keep posting about my [SRG woes](https://delphinqa.ling.washington.edu/t/spanish-clitics-and-freeling-tokenizer-morphological-analyzer/907/2) on Discourse in the meantime :).

**Comment by olzama** [comment]
Do some @delph-in/participants  plan on attending this? @arademaker could we use your zoom again?

**Reply by arademaker** [reply]
Sure! We can use my Zoom or Webex.  Whatever you prefer.

**Reply by olzama** [reply]
Great! Zoom works well :). Thanks, Alex!

**Reply by emilymbender** [reply]
With some luck I'll be able to join this time.

**Comment by danflick** [comment]
Me too.

Dan

**Comment by olzama** [comment]
@suarezallegue Lorena, note this, in case you have nothing better to do on Monday at 19:30 Spanish time :). This is our grammar engineering meeting where we discuss matters related to the kind of grammars and treebanks that you are working with. I will post the Zoom link again on Monday but it should be: https://fgv-br.zoom.us/my/arademaker.

**Reply by suarezallegue** [reply]
I'll log in for sure. Thanks for the invitation, Olga!

**Comment by olzama** [comment]
@delph-in/participants a reminder that we have a grammar engineering meeting today at 18:30 Loondon time, using @arademaker 's zoom:

https://fgv-br.zoom.us/my/arademaker

See some of you there!

**Comment by olzama** [comment]
Technical problems... But Alex and I are here and are trying to start the meeting :).

**Comment by danflick** [comment]
Ready when you are.

 Dan

**Comment by arademaker** [comment]
https://fgv-br.zoom.us/j/93604966743?pwd=NlNKcTBzK093TEQwTkJXSURCM0RPdz09

Please use it

**Comment by olzama** [comment]
Says "authorized participants only"?

**Comment by danflick** [comment]
Me too - it wants me to "sign in".

 Dan

**Comment by emilymbender** [comment]
I saw that too and just signed in to Zoom and it worked.

**Comment by danflick** [comment]
Hmm, it seems to either insist on a password, or, if I try logging in
through Google, it wants a pass code.  Alas.

**Comment by danflick** [comment]
But carry on without me this week.  No worries.

 Dan

**Comment by olzama** [comment]
Noooo Dan don't go! We are working on it!

**Comment by emilymbender** [comment]
New link:
https://washington.zoom.us/j/4560584883
